src/pages/home/components/ClaimModal/claimModal.tsx

```diff
@@ -300,7 +337,7 @@ const ClaimModal = () => {
     <>
       <Modal title={`Claim ${formatWeiBalance(activeChain.maxClaimAmount)} ${activeChain.symbol}`} size="small"
              closeModalHandler={closeClaimModal} isOpen={isOpen}>
-        <ClaimModalBody chain={activeChain}/>
+        <ClaimModalBody chain={activeChain} isFull={activeClaimHistory.length >= 5}/>
```

Comment:
activeClaimHistory despite successful, accepted, and approved claims also contain pending and rejected claim tries. So I guess this line is not correct 100%.